test/CMakeLists.txt

```diff
@@ -1,5 +1,16 @@
 if (BUILD_TESTS)
 
+    cmake_policy(SET CMP0064 NEW)
```

Comment:
https://cmake.org/cmake/help/v3.4/policy/CMP0064.html
The NEW behavior is to interpret the TEST operator.
This policy was introduced in CMake version 3.4.